dbt_subprojects/daily_spellbook/dbt_project.yml

```diff
@@ -349,3 +349,8 @@ models:
       +schema: aerodrome
       base:
         +schema: aerodrome_base
+
+    spaceharpoon:
+      +schema: spaceharpoon
+      polygon:
+        +schema: spaceharpoon_polygon
```

Comment:
these lines are not needed (as we define the schema in the inline config in the models)

Reply:
Noted!